internal/app/machined/pkg/system/services/etcd.go

```diff
@@ -318,12 +318,8 @@ func buildInitialCluster(config runtime.Configurator, name, ip string) (initial
 	return initial, nil
 }
 
-func blacklistError(s string) error {
-	return fmt.Errorf("extra etcd arg %q is not allowed", s)
-}
-
 // nolint: gocyclo
-func args(config runtime.Configurator) ([]string, error) {
+func etcdArgs(config runtime.Configurator) ([]string, error) {
```

Comment:
nit, but would it be better to make them more of methods on `Etcd` type to get them scoped under the service?

Reply:
Yep!